pinot-core/src/test/java/org/apache/pinot/core/query/request/context/utils/BrokerRequestToQueryContextConverterTest.java

```diff
@@ -581,14 +579,12 @@ public void testHardcodedQueries() {
   public void testFilteredAggregations() {
```

Comment:
Can you please add a test where filter inside FILTER clause is on a function and/or combination of identifier and function ?